### ISSUE TITLE
A11y_ML .NET Model Builder_Image Classification-Evaluate_ScreenReader: While navigating from Evaluate tab to Consume tab, Narrator/NVDA is not announcing the tab name.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [dotnet/machinelearning-modelbuilder#1643]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #BM-VisualStudioClient-Win32-Jul21; #A11ySev2; #Visual Studio Client; #WCAG1.3.1; #A11yPreview; #Narrator; #NVDA; #Screen Reader;

### Environment Details:
Microsoft Visual Studio

### Note:
This issue is observed while navigating from Train tab to Evaluate tab and Consume tab to Next Steps tabs, screen reader is not announcing tab name.

### Re-pro Steps:

1. Turn on Narrator/NVDA. Open Visual Studio Enterprise 2022 Preview
2. Create a new .Net Core Console App (C#) ex: WpfApp
3. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
4. Tab till "Image classification" and press enter
5. Select "Local (CPU)" environment and tab till "Next" button and press enter
6. In Data tab, Tab till "Browse" button and select the input file
7. After uploading data, click next step button, start training in Train tab
8. After completing training, click next step button, complete all the steps and navigate to 'Consume' tab
9. Verify whether narrator is reading next tab name when navigated from one tab to other tab

### Actual Result:
While navigating from Evaluate tab to Consume tab, Narrator/NVDA is not announcing the tab name.

### Expected Result:
While navigating from Evaluate tab to Consume tab, Narrator/NVDA should announce Consume tab name clearly. Screen reader should read tab names while navigating from one tab to other.

### User Impact:
Screen reader users will not know to which tab they are navigating to if Narrator/NVDA is not announcing tab names

**Comment by zewditu** [comment]
The same issue also happens when we are navigating from Consume to Next steps tab. The issue is not only in image classification but for all scenarios.

**Comment by zewditu** [comment]
This issue happens when the set focus is set to the first/ default interactive control  b/c in both Consume tab( Copy hyperlink ) and next steps tab(Deploy your model ) the focus is set to the first / default interactive control.  Hence in order to enable the narrator to announce tab names, we set focus to non-default controls / first interactive control. This may need a design change.

**Comment by zewditu** [comment]
Action Item: contact the Accessibility team.

**Comment by zewditu** [comment]
After accessibility team office hour there is a follow-up email with Scott O'Hara to answer offline.

**Comment by zewditu** [comment]
I had a call with Scott O'Hara on teams, what he updates me is this is WPF by design, not a  Model Builder bug.

**Comment by zewditu** [comment]
I confirmed that this is how the narrator announce by default and the UIA tree shows that everything is set correctly ![image.png](https://images.zenhubusercontent.com/5d014b8c2b5aa11c6166d13d/1a82b8ff-33f0-4c7c-b627-8ca73cddb31e)

![image.png](https://images.zenhubusercontent.com/5d014b8c2b5aa11c6166d13d/f9a3fb9d-d7c9-47fc-bd49-58b6feae6919)

**Comment by beccamc** [comment]
@NiharikaTipirishetty This is by design. As you can see in the UIA tree above, we have correctly set the tab name. Narrator is choosing to announce the interactable element instead. The difference between The Consume and Next Step pages and the previous pages is which element has focus on first load. When the focus is on the first interactable element, narrator chooses to announce that instead of the tab name. 

If this is a problem, it is with narrator. Thanks!

**Comment by NiharikaTipirishetty** [comment]
Issue is fixed with NVDA. While navigating from Evaluate tab to Consume tab, NVDA is announcing the tab name.

![1643_NVDA](https://user-images.githubusercontent.com/83570638/135086583-ab7df6d3-6ea3-437f-a380-c0d0a2be1940.png)

**Comment by NiharikaTipirishetty** [comment]
@beccamc Since the issue is still reproducing with Narrator, could you please provide sample app where similar issue reproduces with Narrator to raise an external bug against Narrator

**Comment by beccamc** [comment]
@NiharikaTipirishetty I will work on getting the sample app. Can the priority of this bug be lowered since it is a Narrator problem?

**Comment by beccamc** [comment]
I've created a small app that tries to demonstrate the problem. It is attached. Here is a video of the app: 

https://user-images.githubusercontent.com/9122518/135358398-34fb7d98-730f-4589-943f-c31ea1767ed0.mp4

[WPF_VSIX_Repo.zip](https://github.com/dotnet/machinelearning-modelbuilder/files/7255410/WPF_VSIX_Repo.zip)

@NiharikaTipirishetty Is this sufficient to close this bug? Thanks!

**Comment by NiharikaTipirishetty** [comment]
@beccamc Could you please confirm if this feature is publicly available? If the feature is publicly available we don't need sample app. Once the bug is logged against Narrator we will be closing this bug as it is working fine with NVDA.

**Comment by beccamc** [comment]
@NiharikaTipirishetty, yes the navigation bug from Evaluate to Consume tab is publicly available. It's visible in our public release here https://marketplace.visualstudio.com/items?itemName=MLNET.07

**Comment by NiharikaTipirishetty** [comment]
Logged an external bug against Narrator. Since the issue is not repro with NVDA screen reader, closing this bug
External Narrator bug: https://microsoft.visualstudio.com/OS/_workitems/edit/36546211

No issue with NVDA:
![1643_NVDA_no issue](https://user-images.githubusercontent.com/83570638/135466286-39feedcf-84d5-4d27-a80c-7110eeb68b79.png)

**Comment by beccamc** [comment]
Thank you @NiharikaTipirishetty!

**Comment by akashelijah05** [comment]
#Closed;
